app/views/fragments/productLists/productListUk.scala.html

```diff
@@ -1,5 +1,5 @@
 @import model.DigitalEdition.UK
-@import services.BlackFriday._
+@import services.FlashSale._
 @import model.promoCodes._
 
 @(promoCodes: Map[PromoCodeKey, String], includeEditionInUrl: Boolean = false)
```

Comment:
Some of this copy was being removed here: https://github.com/guardian/subscriptions-frontend/pull/1023/files#diff-fbdc75d5de16bcd9a56b38da9aa67504

If this sale only applies to Digital Pack, then I think this still needs to go?

Reply:
Yes I just realised that doh! Good spot Jacob.